tests/test_logging_util.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+import pytest
+from great_expectations.compatibility.pydantic import (
+    BaseModel,
+    ValidationError,
+)
+
+from great_expectations_cloud.logging.logging_util import generate_validation_error_text
+
+
+class LoggingUtilTestModel(BaseModel):
+    string_one: str
+    string_two: str
+
+
+@pytest.fixture
+def validation_error_missing_string_one():
+    try:
+        LoggingUtilTestModel(string_one=None, string_two="i am string two")
+    except ValidationError as validation_error:
+        return validation_error
```

Comment:
In this method I'm forcing a validation error from pydantic. This way if the pydantic model structure for their ValidationErrors ever changes, this test will fail and we can rewrite the way we pull the missing attribute names from the error.